wavelength-plugins/factions.js

```diff
@@ -673,7 +673,7 @@ exports.commands = {
 			factions[factionid].ranks[toId(rank)].users.push(targetUser.userid);
 			write();
 			rank = factions[factionid].ranks[toId(rank)].title;
-			targetUser.send("|popup||html|" + user.name + " has set your faction rank in " + Chat.escapeHTML(factions[factionid].name) + " to " + Chat.escapeHTML(rank) + ".");
+			targetUser.send("|popup||html|" + WL.nameColor(user.name) + " has set your faction rank in " + Chat.escapeHTML(factions[factionid].name) + " to " + Chat.escapeHTML(rank) + ".");
```

Comment:
Usernames should be HTML escaped in HTML outputs.

Reply:
Wait WL.nameColor = html?

Reply:
So why "escape"

Reply:
`Wl.nameColor(Chat.escapeHTML(user.name))`

Reply:
This is so that usernames with HTML charachters dont cause issues

Reply:
Yea